### ISSUE TITLE
Feature Request: conversion at specific date

### ISSUE THREAD

**Issue by uwe-schwarz** [chubin/rate.sx#37]
Accept a date (range) for specific values: https://rate.sx/0.1btc@2021-03-01

Expected answer would be the conversion of 0.1btc to usd with the end-price of that day ($49672 (Mon 01 23:50)) -> $4967.2.

**Comment by chubin** [comment]
Yes, this is a cool idea; as well as the whole `rate.sx` output, like:

```
curl rate.sx/@DATE
```
